app/src/main/java/com/poupa/vinylmusicplayer/adapter/SearchAdapter.java

```diff
@@ -74,33 +84,64 @@ public void onBindViewHolder(@NonNull final ViewHolder holder, int position) {
                         .asDrawable()
                         .load(VinylGlideExtension.getSongModel(album.safeGetFirstSong()))
                         .transition(VinylGlideExtension.getDefaultTransition())
-                        .songOptions(album.safeGetFirstSong())
+                        .songOptions(album.safeGetFirstSong()), activity)
                         .into(holder.image);
                 break;
             case ARTIST:
                 final Artist artist = (Artist) dataSet.get(position);
                 holder.title.setText(artist.getName());
                 holder.text.setText(MusicUtil.getArtistInfoString(activity, artist));
-                GlideApp.with(activity)
-                        .asBitmap()
-                        .load(VinylGlideExtension.getArtistModel(artist))
-                        .transition(VinylGlideExtension.getDefaultTransition())
-                        .artistOptions(artist)
-                        .into(holder.image);
+                if (PreferenceUtil.getInstance().useMaterialDesign()) {
+                    GlideApp.with(activity)
+                            .asBitmap()
+                            .load(VinylGlideExtension.getArtistModel(artist))
+                            .transition(VinylGlideExtension.getDefaultTransition())
+                            .circleCrop()
+                            .artistOptions(artist)
+                            .into(holder.image);
+                } else {
+                    GlideApp.with(activity)
+                            .asBitmap()
+                            .load(VinylGlideExtension.getArtistModel(artist))
+                            .transition(VinylGlideExtension.getDefaultTransition())
+                            .artistOptions(artist)
+                            .into(holder.image);
+                }
                 break;
             case SONG:
                 final Song song = (Song) dataSet.get(position);
                 holder.title.setText(song.title);
                 holder.text.setText(MusicUtil.getSongInfoString(song));
-
-                PlayingSongDecorationUtil.decorate(holder.title, holder.image, holder.imageText, song, activity, false);
+                if (PreferenceUtil.getInstance().useMaterialDesign()) {
+                    GlideApp.with(activity)
+                            .asBitmap()
+                            .load(VinylGlideExtension.getSongModel(song))
+                            .transition(VinylGlideExtension.getDefaultTransition())
+                            .apply(RequestOptions.bitmapTransform(new RoundedCorners(
+                                    activity.getResources().getInteger(R.integer.rounded_corner))))
+                            .songOptions(song)
+                            .into(holder.image);
+
+                    PlayingSongDecorationUtil.decorate(holder.title, holder.image, holder.imageText, song, activity, true);
```

Comment:
Replace true/false by `PreferenceUtil.getInstance().useMaterialDesign()` and move the line out of the conditional block